CHANGELOG.rst

```diff
@@ -1,5 +1,24 @@
 Changelog
 =========
+1.16.0 (2021-04-22)
+-------------------
+MLflow 1.16.0 includes several major features and improvements:
+
+Features:
+
+- Add PySpark ML autologging for basic estimators (e.g. LinearRegression) (#4228, @WeichenXu123)
+- Add CatBoost model persistence APIs (#2417, @harupy)
```

Comment:
- Add ``mlflow.catboost.log_model``, ``mlflow.catboost.save_model``, ``mlflow.catboost.load_model`` APIs for CatBoost model persistence